04-brute-force-and-exhaustive-search/1759/zeroChoi.py

```diff
@@ -0,0 +1,40 @@
+vowel = ['a', 'e', 'i', 'o', 'u']
+
+L, C = map(int, input().split())
+words = input().split()
+
+words.sort()
+
+
+def check(arr):
+    v_count, c_count = 0, 0  # 모음 개수, 자음 개수
+
+    for i in arr:
+        if i in vowel:
+            v_count += 1
+        else:
+            c_count += 1
+
+    if v_count >= 1 and c_count >= 2:
+        return True
+    else:
+        return False
```

Comment:
조건문의 결과 자체를 반환해도 좋을 것 같네요!
```suggestion
    return v_count >= 1 and c_count >= 2:
```